File: recipe-site/app/_models/Weapon.ts
export default class Weapon {
    weaponId: number;
    characterId: number;
    name: string;
    damage: string;
    description: string;

    constructor(weaponId: number, characterId: number, name: string, damage: string, description: string) {
        this.weaponId = weaponId;
        this.characterId = characterId;
        this.name = name;
        this.damage = damage;
        this.description = description;
    }
}

File: java-recipes/src/main/java/com/fargopolis/models/Weapon.java
package com.fargopolis.models;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class Weapon {
    private Integer weaponId;
    private Integer characterId;
    private String name;
    private String damage;
    private String description;
}
